packages/manager/modules/bm-server-components/src/server/server.controller.js

```diff
@@ -663,7 +657,7 @@ export default class ServerCtrl {
   }
 
   getTabItemUrl(tabItemName) {
-    return this.$state.href(`app.dedicated-server.server.${tabItemName}`);
+    return this.$state.href(`${this.statePrefix}.${tabItemName}`);
```

Comment:
Do you still need `statePrefix` ?

Reply:
Yes without doesn't work because it isn't child.